pkg/kubernetes/upgrade.go

```diff
@@ -66,7 +66,6 @@ func Upgrade(conf UpgradeConfig) error {
 	}
 
 	upgradeClient := helm.NewUpgrade(helmConf)
-	upgradeClient.ResetValues = true
```

Comment:
> > If RestValues flag is not set, it will not apply the new values in the new chart during upgrade command run. But isn't that what we want?
> 
> In this PR, the ResetValues flag is set to `true`, meaning it will applies new values in the chart, which is what we want.

I think I am missing something here. This PR removes the line which sets the Reset Values to "true" ?

Reply:
I meant ReuseValues, and apparently I didn't push it.. until now.